### PR TITLE
Fix Alix40 layout

### PR DESCRIPTION
**Pull request by Rossmallow**
## Description
Update the layout so that the keys aren't displayed on top of each other

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* [12390](https://github.com/qmk/qmk_firmware/issues/12390)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/tokyokeyboard/alix40/info.json [hunk 1]
```diff
@@ -163,27 +163,27 @@
...
             {
                 "label": ".",
-                "x": 7.25,
+                "x": 11.25,
                 "y": 2
             },
             {
                 "label": "/",
-                "x": 7.25,
+                "x": 126.25,
```
Comment (ridingqwerty) [inline keyboards/tokyokeyboard/alix40/info.json:186]: Is `126.25` correct here?  Seems rather large.

Comment (Rossmallow) [reply keyboards/tokyokeyboard/alix40/info.json:186]: You're right. That's a typo. It should be 12.25

**Review by ridingqwerty — APPROVED** [review]
Looks good to me.

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**